diff --git a/app/(auth)/signUp/_components/Signup.tsx b/app/(auth)/signUp/_components/Signup.tsx
index 2652f2f..2d9f6d5 100644
--- a/app/(auth)/signUp/_components/Signup.tsx
+++ b/app/(auth)/signUp/_components/Signup.tsx
@@ -37,6 +37,17 @@ const SESSION_KEYS = {
   REF_CODE: "optisage_ref_code",
 } as const;
 
+export const amazonAuthUrl = `https://sellercentral.amazon.com/apps/authorize/consent?${new URLSearchParams(
+    {
+      application_id: process.env.NEXT_PUBLIC_AMAZON_CLIENT_ID!,
+      state: "ourauth",
+      version: "beta",
+      response_type: "code",
+      scope: "sellingpartnerapi::authorization",
+      redirect_uri: process.env.NEXT_PUBLIC_AMAZON_REDIRECT_URI!,
+    }
+  ).toString()}`;
+
 const Signup = () => {
   const router = useRouter();
   const searchParams = useSearchParams();
@@ -126,16 +137,7 @@ const [subscribe, { isLoading }] = useSignupMutation();
     passwordFormik.setFieldValue('confirmPassword', form.confirmPassword);
   }, [form.password, form.confirmPassword]);
 
-  const amazonAuthUrl = `https://sellercentral.amazon.com/apps/authorize/consent?${new URLSearchParams(
-    {
-      application_id: process.env.NEXT_PUBLIC_AMAZON_CLIENT_ID!,
-      state: "ourauth",
-      version: "beta",
-      response_type: "code",
-      scope: "sellingpartnerapi::authorization",
-      redirect_uri: process.env.NEXT_PUBLIC_AMAZON_REDIRECT_URI!,
-    }
-  ).toString()}`;
+  
 
   // Helper functions for session storage
   const saveToSessionStorage = (key: string, value: any) => {
@@ -916,7 +918,6 @@ const [subscribe, { isLoading }] = useSignupMutation();
 
       <Link href={amazonAuthUrl} target="_blank">
         <button className="w-full border border-[#EDEDED] px-5 py-3 rounded-[10px] flex items-center gap-4 hover:bg-gray-50 transition-colors duration-200 mb-4">
-         
           <Image
             src="https://avatar.iran.liara.run/public/38"
             alt="Avatar"
@@ -933,10 +934,7 @@ const [subscribe, { isLoading }] = useSignupMutation();
             <span className="bg-[#FFC56E66] rounded-full py-1 px-3 text-xs text-[#7E5806] mt-1.5">
               Login to Connect Store
             </span>
-           
-           
           </span>
-           <span className=" text-xs font-semibold">Optional</span>
 
           <HiMiniArrowLongRight className="size-5 text-[#0F172A]" />
         </button>
