diff --git a/redux/store.ts b/redux/store.ts
index 227eed3..c0bc79e 100644
--- a/redux/store.ts
+++ b/redux/store.ts
@@ -1,6 +1,7 @@
 import { combineReducers, configureStore } from "@reduxjs/toolkit";
 import authReducer from "./slice/authSlice";
 import globalReducer from "./slice/globalSlice";
+import chatReducer from "./slice/chatSlice"; // Add this import
 import { authApi } from "./api/auth";
 import { subscriptionApi } from "./api/subscriptionApi";
 import { productsApi } from "./api/productsApi";
@@ -9,24 +10,31 @@ import storage from "redux-persist/lib/storage";
 import { userApi } from "./api/user";
 import { sellerApi } from "./api/sellerApi";
 import { quickSearchApi } from "./api/quickSearchApi";
+import { keepaApi } from "./api/keepa";
+import { monitorApi } from "./api/monitorApi";
+import { totanApi } from "./api/totanAi";
 
 
 const rootReducer = combineReducers({
   api: authReducer,
   global: globalReducer,
+  chat: chatReducer, // Add the chat reducer
   [authApi.reducerPath]: authApi.reducer,
   [userApi.reducerPath]: userApi.reducer,
   [subscriptionApi.reducerPath]: subscriptionApi.reducer,
   [productsApi.reducerPath]: productsApi.reducer,
   [sellerApi.reducerPath]: sellerApi.reducer,
   [quickSearchApi.reducerPath]: quickSearchApi.reducer,
+  [keepaApi.reducerPath]: keepaApi.reducer,
+  [monitorApi.reducerPath]: monitorApi.reducer,
+  [totanApi.reducerPath]: totanApi.reducer
 
 });
 
 const persistConfig = {
   key: "root",
   storage,
-  whitelist: ["api", "global"], // Whitelist the reducers you want to persist
+  whitelist: ["api", "global", "chat"], // Add "chat" to persist chat history
 };
 const persistedReducer = persistReducer(persistConfig, rootReducer);
 
@@ -42,7 +50,10 @@ export const store = configureStore({
       userApi.middleware,
       productsApi.middleware,
       sellerApi.middleware,
-      quickSearchApi.middleware
+      quickSearchApi.middleware,
+      keepaApi.middleware,
+      monitorApi.middleware,
+      totanApi.middleware
     ]),
 });
 
